src/com/dre/brewery/listeners/Compat1_9.java

```diff
@@ -20,12 +22,10 @@ public void onPlayerDrink(PlayerItemConsumeEvent event) {
 		}
 
 		if (item.getType() == Material.POTION) {
-			try {
-				PotionMeta meta = (PotionMeta) item.getItemMeta();
-				// Throw away former "base" effect and replace with MUNDANE.
-				meta.setBasePotionData(new PotionData(PotionType.MUNDANE, false, false));
-				item.setItemMeta(meta);
-			} catch (Exception exception) {}
+			PotionMeta meta = (PotionMeta) item.getItemMeta();
+			// Throw away former "base" effect and replace with MUNDANE.
+			meta.setBasePotionData(new PotionData(PotionType.MUNDANE, false, false));
```

Comment:
@Sataniel98 This appeared to do the trick! I haven't tested it with a ton of different brew types yet, however.